parm/config/gefs/config.wave

```diff
@@ -19,25 +19,11 @@ export RUNRSTwave="gdas"
 export waveGRDN='1'                   # grid number for ww3_multi
 export waveGRDG='10'                  # grid group for ww3_multi
 export USE_WAV_RMP='NO'               # YES/NO rmp grid remapping pre-processed coefficients
```

Comment:
This is for multi... this should be removed + where this variable is used in ush/forecast_predet.sh can also be removed.